storage_integration_test.go

```diff
@@ -0,0 +1,115 @@
+// +build integration
+
+package tprstorage
+
+/*
+	Usage:
+
+		go test -tags=integration $(glide novendor) [FLAGS]
+
+	Flags:
+
+		-integration.ca string
+			CA file path (default "$HOME/.minikube/ca.crt")
+		-integration.crt string
+			certificate file path (default "$HOME/.minikube/apiserver.crt")
+		-integration.key string
+			key file path (default "$HOME/.minikube/apiserver.key")
+		-integration.server string
```

Comment:
Normally I'd do that, but this is only for testing.

I wanted to make it quick, and not pull additional library to make test flags with double dashes. Also there are some `-test.X` flags that are defined by default so it conforms to the pattern.